### ISSUE TITLE
Why is the config in effnetv2_configs.py not consistent with that in paper?

### ISSUE THREAD

**Issue by HydrogenSulfate** [google/automl#1175]
for example:
the max value of `dropout_rate` and `ram` is 0.2 and 10 in `effnetv2_configs.py`, but 0.3 and 15 in paper, and `dropout_rate` seems not progressively changed in 4 traning stages
```python
'efficientnetv2-s':  # 83.9% @ 22M
        (v2_s_block, 1.0, 1.0, 300, 384, 0.2, 10, 0, 'randaug'),
```
![image](https://user-images.githubusercontent.com/23737287/201473150-104efaed-58bd-47f7-8100-4c00cbf88419.png)

**Comment by Royyyy0** [comment]
I got the same question as yours, do you have any ideas now? Thx

**Comment by HydrogenSulfate** [comment]
> I got the same question as yours, do you have any ideas now? Thx

We've tries several times for reproducing with paddlepaddle, but got many result up to Top1=83.58%, still has 0.27% diff to official Top1=83.85%. 

We find this is overfitting, and we add a `Mixup` and finally achieve to 83.84%, please check the model and config is [efficientnet_v2.py](https://github.com/PaddlePaddle/PaddleClas/blob/develop/ppcls/arch/backbone/model_zoo/efficientnet_v2.py)
[EfficientNetV2_S.yaml](https://github.com/PaddlePaddle/PaddleClas/blob/f5a7461a3b3676f7591b8e5b4a6635f4d3fa5993/ppcls/configs/ImageNet/EfficientNetV2/EfficientNetV2_S.yaml)

The reson might be framework different between tf and paddlepaddle/pytorch. As we all know, many paper in tensorflow always hard to reproduce same metric by other DL frameworks and always got slightly lower metric.
